bin/start

```diff
@@ -54,18 +56,24 @@ done
 
 export RUN_IN_DOCKER
 export CONJUR_DEPLOYMENT
-
-source test/bootstrap.env
+export DEV
+export ENV_RELOAD
 
 # summon environment variable
 export CONJUR_MAJOR_VERSION=4
 
-cd test
-if [[ $DEMO = true ]]; then
-  runScriptWithSummon ./run_demo.sh
-elif [[ $RUN_IN_DOCKER = true ]]; then
+source bootstrap.env
+
+pushd deploy
```

Comment:
1. don't we need to `popd` at the end?
2. please indent everything inside `pushd`